language/move-prover/tests/sources/functional/emits.exp

```diff
@@ -0,0 +1,52 @@
+Move prover returns: exiting with boogie verification errors
+error: function does not emit the event
+
+    ┌── tests/sources/functional/emits.move:62:9 ───
+    │
+ 62 │         emits DummyEvent{msg: 0} to handle;
```

Comment:
What happens right now if there are more events emitted than specified? Can we also say that no events are emitted?

Reply:
Prover does not complain although there are more emitted events than specified ones. It's the next step to verify that no events are emitted other than the specified ones.